peps/pep-0738.rst

```diff
@@ -0,0 +1,351 @@
+
+
+CI resources
+------------
+
+`Anaconda <https://anaconda.com>`__ has offered to provide physical hardware to
+run Android buildbots.
+
+GitHub Actions is able to host Android emulators on their Linux and macOS
+runners. The free tier currently only provides x86_64 machines; however ARM64
+macOS runners `recently became available on paid plans <https://github.blog/
+2023-10-02-introducing-the-new-apple-silicon-powered-m1-macos-larger-runner-for-github-actions/>`__.
+
+
+Packaging
+---------
+
+Android wheels will use tags in the format ``android_<api-level>_<abi>``. For
+example:
+
+* ``android_23_arm64_v8a``
+* ``android_23_x86_64``
+
+For the meaning of ``<api-level>``, see `Version numbering`_. In the context of
+the wheel tag, it indicates the minimum Android version that was selected when
+the wheel was compiled. Installation tools such as pip should interpret this in
+a similar way to the existing macOS tags, i.e. an app with a minimum API level
+of N can incorporate wheels tagged with API level N or older.
+
+This format originates from the Chaquopy project, which currently maintains a
+`wheel repository <https://chaquo.com/pypi-13.1/>`__ with tags varying between
+API levels 16 and 21.
+
+However, relying on a small group of Android enthusiasts to build the whole
+Python ecosystem is not a scalable solution. Until prominent libraries routinely
+release their own Android wheels on PyPI, the ability of the community to adopt
+Python on Android will be limited.
+
+Therefore, it will be necessary to clearly document how projects can add Android
+builds to their CI and release tooling. Adding Android support to tools like
+`crossenv <https://crossenv.readthedocs.io/>`__ and `cibuildwheel
+<https://cibuildwheel.readthedocs.io/>`__ may be one way to achieve this.
+
+
+PEP 11 Update
+-------------
+
+:pep:`11` will be updated to include the two supported Android ABIs. Autoconf
+already identifies them with the following triplets:
+
+* ``aarch64-linux-android``
+* ``x86_64-linux-android``
+
+Petr Viktorin will serve as the initial core team contact for these ABIs.
+
+
+Backwards Compatibility
+=======================
+
+Adding a new platform does not introduce any backwards compatibility concerns to
+CPython itself. However, there may be some backwards compatibility implications
+on the projects that have historically provided CPython support (i.e., BeeWare
+and Kivy) if the final form of any CPython patches don't align with the patches
+they have historically used.
+
+
+Security Implications
+=====================
+
+Adding a new platform does not add any security implications.
+
+
+How to Teach This
+=================
+
+The education needs related to this PEP mostly relate to how developers can
+build Python into an Android app and use it at runtime. Automating and
+documenting this will be the responsibility of higher-level tools such as
+BeeWare's `Chaquopy <https://chaquo.com/chaquopy/>`__ and Kivy's `Buildozer
```

Comment:
Again, worth identifying BeeWare and Chaquopy separately here - At least to my mind, BeeWare is the "I want to write Python and nothing else" approach, and Chaquopy is the "I've got an Android app and I want to use Python" approach - both of which are valid use cases and will likely need to be documented.

Reply:
Done, and I've also mentioned Briefcase.